src/commands/lambda/constants.ts

```diff
@@ -29,29 +31,44 @@ export enum RuntimeType {
   NODE,
   PYTHON,
   RUBY,
+  GO,
 }
 
-export const RUNTIME_LOOKUP = {
+export const RUNTIME_LOOKUP: Record<Runtime, RuntimeType> = {
```

Comment:
We shouldn't be adding previous runtimes, since we don't have support for those who were not previously listed here.

We constantly keep removing runtimes which are out of support for AWS Lambda, so we need to revert this change.

Reply:
Oh ok, thanks for the context 😄